Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocAutoCompleteReferenceValue.css

```diff
@@ -10,6 +10,11 @@ span.bocAutoCompleteReferenceValue
   min-height: 1.7em;
 }
 
+.mozilla span.bocAutoCompleteReferenceValueInput
```

Comment:
please move down to the other `...Input` entries.

I'll also need to understand the use of margin vs. width vs. padding here. When dealing with inline, margin has very specific semantics and we haven't used it with input elements before so I need to understand why this works compared to other approaches.